cmd/query.go

```diff
@@ -45,6 +48,11 @@ var queryCmd = &cli.Command{
 			Aliases: []string{"d"},
 			Usage:   "a delegation allowing the indexer to fetch content from the space",
 		},
+		&cli.BoolFlag{
+			Name:    "telemetry",
+			Usage:   "propagate tracing context on query requests",
+			EnvVars: []string{"CLIENT_TELEMETRY_ENABLED"},
```

Comment:
nit: you could consider prefixing the env var with something like `INDEXER_`. As is I'd say the chance of collision is not super high, but it's there 😄.